tests/hpa_test.yaml

```diff
@@ -1,16 +1,21 @@
 ---
 suite: test horizontal pod autoscaler
-
 templates:
   - hpa.yaml
-
 set:
   autoscaling.enabled: true
-
+  nameOverride: my-chart-name
```

Comment:
Set this inside the test, we need to only validate this is working, other test should use the default value

Reply:
okay